changelogs/fragments/892-vmware_vmkernel.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmware_vmkernel - manage vSAN service configuration on vmk interfaces so that it is possible to define multiple vSAN interfaces at the same, along with merely explicitly adding/removing interfaces, instead of applying a configuration that only enables the last specified vmk interface (https://github.com/ansible-collections/community.vmware/issues/891)
```

Comment:
This feels quite wordy to me. Do you think we could shorten this a bit? `Fix a bug that prevented enabling VSAN on more then one vmk and crashed the module when trying to disable VSAN (https://github.com/ansible-collections/community.vmware/issues/891)` or similar.

Reply:
OK, I shortened it.